cranelift/codegen/meta/src/isa/x86/opcodes.rs

```diff
@@ -665,6 +665,10 @@ pub static UCOMISS: [u8; 2] = [0x0f, 0x2e];
 /// Raise invalid opcode instruction.
 pub static UNDEFINED2: [u8; 2] = [0x0f, 0x0b];
 
+/// Convert four packed unsigned doubleword integers from xmm2/m128/m32bcst to packed
+/// single-precision floating-point values in xmm1 with writemask k1 (AVX512VL, AVX512F).
+pub static VCVTUDQ2PS: [u8; 3] = [0xf2, 0x0f, 0x7a];
```

Comment:
What rounding behavior does `VCVTUDQ2PS` have?  Is it: +inf -inf to-zero to-nearest or is it mxcsr dependent?  Please mention this in the comment.

Reply:
Documented in this description.